.github/workflows/beagle-bone-blue-binaries.yml

```diff
@@ -0,0 +1,101 @@
+name: beagle-bone-blue-binaries
+on:
+  push:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '30 5,17 * * *'
+jobs:
+  latest:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      #- run: sudo apt update
+      #- run: sudo apt install git
+      - run: git clone https://github.com/ardupilot/ardupilot.git
+      - run: cd ardupilot && ./Tools/environment_install/install-prereqs-ubuntu.sh
```

Comment:
see on our Dockerfile how to use install-prereqs-ubuntu.sh with docker ... you don't need everything, specially GUI stuff